login/package.json

```diff
@@ -14,6 +14,6 @@
     "typescript": "^3.5.2"
   },
   "dependencies": {
-    "@actions/core": "^1.0.0"
+    "@actions/core": "^1.2.6"
```

Comment:
@ds-ms , once we remove the login Action from this repo, we should be able to remove this dependency. But till then  we can merge this change, any validation we need for this Action?

Reply:
Sure

Reply:
We need to add the upgraded node_module to this repo as well